src/main/java/namor/template/stack/MyOwnException.java

```diff
@@ -0,0 +1,11 @@
+package namor.template.stack;
+
+/**
+ * Created by Roman on 04.04.2017.
+ */
+public class MyOwnException extends RuntimeException {
```

Comment:
No comments. `MyOwnException` should be renamed and never showed to anybody names like this ;)